spec/features/help_page_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'As a vistor' do
+  describe 'When I visit /help' do
+    it 'I see a description of how to use the application' do
+      visit '/help'
+
+      expect(current_path).to eq(help_path)
+
+      expect(page).to have_css('#what-is-gs')
```

Comment:
Verifying the css id is unnecessary when the same id is used to create a `within` block